pkg/dump/schemas.go

```diff
@@ -43,8 +43,7 @@ func NewSchemaFetcher(ctx context.Context, client *kong.Client, isKonnect bool)
 		vaultSchemaCache: types.NewSchemaCache(func(ctx context.Context,
 			vaultType string,
 		) (map[string]interface{}, error) {
-			// works only for gateway, not konnect
```

Comment:
Related question - I see for partials and plugins, we load schema from go-kong - does this work for konnect as well?